### ISSUE TITLE
Improve styling

### ISSUE THREAD

**Issue by VatsKan** [fac19/Week8-BFOP#44]
- styling: log out and create new post underlines or overlines on hover (not sure if this is intentional). 
- would be cool to see styling on edit and delete buttons also.

(NB styling is incredible otherwise! <3)

**Comment by jc2820** [comment]
Agreed on those three points above. There was a lot to get in this week and you've done an impressive job on styling considering. Certainly worth bringing those buttons up to the standard of everything else!
